<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1055</width>
    <height>804</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout" stretch="0,20">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QWidget" name="widget" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout" stretch="10,35">
       <item>
        <widget class="QWidget" name="widget_4" native="true">
         <property name="styleSheet">
          <string notr="true">QLabel{
	
	font: 14pt &quot;黑体&quot;;
}

QPushButton{
	font: 14pt &quot;黑体&quot;;
}</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>COM</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QWidget" name="widget_3" native="true">
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>100</height>
             </size>
            </property>
            <layout class="QGridLayout" name="gridLayout" rowstretch="10,10" columnstretch="5,1" rowminimumheight="10,10" columnminimumwidth="5,1">
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <property name="spacing">
              <number>6</number>
             </property>
             <item row="0" column="0">
              <widget class="QComboBox" name="comboBox">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>36</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>16777215</width>
                 <height>36</height>
                </size>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QPushButton" name="pushButton_PortRefresh">
               <property name="minimumSize">
                <size>
                 <width>48</width>
                 <height>38</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>48</width>
                 <height>38</height>
                </size>
               </property>
               <property name="text">
                <string>刷新</string>
               </property>
              </widget>
             </item>
             <item row="1" column="0" colspan="2">
              <widget class="QPushButton" name="pushButton_PortOpen">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>36</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>16777215</width>
                 <height>36</height>
                </size>
               </property>
               <property name="text">
                <string>连接</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEdit">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>40</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_6">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>40</height>
             </size>
            </property>
            <property name="text">
             <string>监听设备</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_7">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>40</height>
             </size>
            </property>
            <property name="text">
             <string>INIT</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>40</height>
             </size>
            </property>
            <property name="text">
             <string>欠压测试</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_2">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>40</height>
             </size>
            </property>
            <property name="text">
             <string>点火测试</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_3">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>40</height>
             </size>
            </property>
            <property name="text">
             <string>超压测试</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_4">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>40</height>
             </size>
            </property>
            <property name="text">
             <string>提交到服务器</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_5">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>40</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="text">
             <string>Clear</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_8">
            <property name="text">
             <string>PushButton</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>369</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QTabWidget" name="tabWidget">
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="tab">
          <attribute name="title">
           <string>TEST-ITEM</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QTableWidget" name="tableWidget">
             <property name="rowCount">
              <number>8</number>
             </property>
             <property name="columnCount">
              <number>4</number>
             </property>
             <row/>
             <row/>
             <row/>
             <row/>
             <row/>
             <row/>
             <row/>
             <row/>
             <column>
              <property name="text">
               <string>SN号</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>欠压</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>超压</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>点火开阀</string>
              </property>
             </column>
             <item row="0" column="0">
              <property name="text">
               <string> </string>
              </property>
             </item>
             <item row="0" column="1">
              <property name="text">
               <string> </string>
              </property>
             </item>
             <item row="0" column="2">
              <property name="text">
               <string> </string>
              </property>
             </item>
             <item row="0" column="3">
              <property name="text">
               <string> </string>
              </property>
             </item>
             <item row="1" column="0">
              <property name="text">
               <string> </string>
              </property>
             </item>
             <item row="1" column="1">
              <property name="text">
               <string> </string>
              </property>
             </item>
             <item row="1" column="2">
              <property name="text">
               <string> </string>
              </property>
             </item>
             <item row="1" column="3">
              <property name="text">
               <string> </string>
              </property>
             </item>
             <item row="2" column="0">
              <property name="text">
               <string> </string>
              </property>
             </item>
             <item row="2" column="1">
              <property name="text">
               <string> </string>
              </property>
             </item>
             <item row="2" column="2">
              <property name="text">
               <string> </string>
              </property>
             </item>
             <item row="2" column="3">
              <property name="text">
               <string> </string>
              </property>
             </item>
             <item row="3" column="0">
              <property name="text">
               <string> </string>
              </property>
             </item>
             <item row="3" column="1">
              <property name="text">
               <string/>
              </property>
             </item>
             <item row="3" column="2">
              <property name="text">
               <string/>
              </property>
             </item>
             <item row="3" column="3">
              <property name="text">
               <string/>
              </property>
             </item>
             <item row="4" column="0">
              <property name="text">
               <string> </string>
              </property>
             </item>
             <item row="4" column="1">
              <property name="text">
               <string/>
              </property>
             </item>
             <item row="4" column="2">
              <property name="text">
               <string/>
              </property>
             </item>
             <item row="4" column="3">
              <property name="text">
               <string/>
              </property>
             </item>
             <item row="5" column="0">
              <property name="text">
               <string> </string>
              </property>
             </item>
             <item row="5" column="1">
              <property name="text">
               <string/>
              </property>
             </item>
             <item row="5" column="2">
              <property name="text">
               <string/>
              </property>
             </item>
             <item row="5" column="3">
              <property name="text">
               <string/>
              </property>
             </item>
             <item row="6" column="0">
              <property name="text">
               <string> </string>
              </property>
             </item>
             <item row="6" column="1">
              <property name="text">
               <string/>
              </property>
             </item>
             <item row="6" column="2">
              <property name="text">
               <string/>
              </property>
             </item>
             <item row="6" column="3">
              <property name="text">
               <string/>
              </property>
             </item>
             <item row="7" column="0">
              <property name="text">
               <string/>
              </property>
             </item>
             <item row="7" column="1">
              <property name="text">
               <string/>
              </property>
             </item>
             <item row="7" column="2">
              <property name="text">
               <string/>
              </property>
             </item>
             <item row="7" column="3">
              <property name="text">
               <string/>
              </property>
             </item>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="tab_2">
          <attribute name="title">
           <string>logs</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QWidget" name="widget_5" native="true">
             <layout class="QVBoxLayout" name="verticalLayout_5">
              <property name="leftMargin">
               <number>0</number>
              </property>
              <property name="topMargin">
               <number>0</number>
              </property>
              <property name="rightMargin">
               <number>0</number>
              </property>
              <property name="bottomMargin">
               <number>0</number>
              </property>
              <item>
               <widget class="QTextBrowser" name="textBrowser"/>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widget_2" native="true">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>30</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>36</height>
       </size>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1055</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menufile">
    <property name="title">
     <string>file</string>
    </property>
    <addaction name="actionlogs"/>
    <addaction name="actionxmp"/>
   </widget>
   <widget class="QMenu" name="menusettings">
    <property name="title">
     <string>settings</string>
    </property>
    <addaction name="actionconfig"/>
    <addaction name="actionreset"/>
   </widget>
   <addaction name="menufile"/>
   <addaction name="menusettings"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionlogs">
   <property name="text">
    <string>logs</string>
   </property>
  </action>
  <action name="actionxmp">
   <property name="text">
    <string>xml</string>
   </property>
  </action>
  <action name="actionconfig">
   <property name="text">
    <string>config</string>
   </property>
  </action>
  <action name="actionreset">
   <property name="text">
    <string>reset</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
